eiffel-vocabulary/EiffelServiceStartedEvent.md

```diff
@@ -0,0 +1,21 @@
+
+# EiffelSeriveStartedEvent (ArtS)
+
+The EiffelServiceStartedEvent declares that a service has been started and is ready for use.
```

Comment:
Clarified some more...
____Description:__ The EiffelServiceStartedEvent declares that a service contained within a defined environment (declared by EiffelEnvironmentDefinedEvent) been started successfully and is ready for use.